src/main/java/io/vertx/ext/mongo/package-info.java

```diff
@@ -253,6 +253,17 @@
  * {@link examples.Examples#example12}
  * ----
  *
+ * === MongoDB Extended JSON support
+ *
+ * For now, only date type is supported (cf http://docs.mongodb.org/manual/reference/mongodb-extended-json )
+ *
+ * Here's an example of inserting a document with a date field
```

Comment:
I'm guessing that the date long is like a Java date representation i.e. number of millis since Jan 1 1970 (?) but I think we should be explicit about this in the documentation...